Lib/test/test_imghdr.py

```diff
@@ -77,13 +77,13 @@ def test_file_pos(self):
             self.assertEqual(stream.tell(), pos)
 
     def test_bad_args(self):
-        with self.assertRaises(TypeError):
+        with self.assertRaises(ValueError):
```

Comment:
Revert to be consistent with above.
```suggestion
        with self.assertRaises(TypeError):
```